README.md

```diff
@@ -10,10 +10,8 @@ Each service is an NPM package.
 
 ## Installation
 
-From root run: 
-<pre><code>npm i</code></pre>
-From each service root run: 
-<pre><code>npm i</code></pre>
+From root run: `npm i`
+From each service root run: `npm i`
```

Comment:
Wouldn't the Makefile do a npm install in each service for us, rather than needing to run manually?

Reply:
Thank you, made another update.